src/cmd/domain.go

```diff
@@ -0,0 +1,233 @@
+package cmd
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"github.com/creasty/defaults"
+	"github.com/opslevel/cli/common"
+	"github.com/opslevel/opslevel-go/v2023"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"os"
+	"strings"
+)
+
+var assignSystemCmd = &cobra.Command{
+	Use:        "system ID|ALIAS SYSTEM",
+	Short:      "Add a system to a domain",
+	Example:    `opslevel create system my_domain my_system`,
+	Args:       cobra.ExactArgs(2),
+	ArgAliases: []string{"DOMAIN_ID", "DOMAIN_ALIAS", "SYSTEM"},
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		system := args[1]
+
+		domain, err := getClientGQL().GetDomain(key)
+		cobra.CheckErr(err)
+		common.WasFound(domain.Id == "", key)
+
+		addErr := domain.AssignSystem(getClientGQL(), system)
+		cobra.CheckErr(addErr)
+		fmt.Printf("add system '%s' to domain '%s'\n", system, domain.Name)
+	},
+}
+
+var createDomainCmd = &cobra.Command{
+	Use:   "domain",
+	Short: "Create a domain",
+	Long: `Create a domain
+
+cat << EOF | opslevel create domain -f -
+name: "My Domain"
+description: "Hello World Domain"
+ownerId: "Z2lkOi8vb3BzbGV2ZWwvVGVhbS83NjY"
```

Comment:
🤮 - but I understand why.  I wonder if there is a better way for us to cover this up with the CLI?  Maybe with a special flag on the command `opslevel create domain --ownerType=group` or something?  Then the field can be an alias and we can lookup the group and fill it with the id.

Reply:
I didn't like this either but wasn't sure if there was an easy way to convert this?  Could we write something on the opslevel-go side that gets the team with the alias and then passes the ID to the input?